ci-operator/step-registry/opendatahub-io/ci/s390x-build/build-publish/opendatahub-io-ci-s390x-build-build-publish-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: opendatahub-io-ci-s390x-build-build-publish
+  documentation: |-
+    This workflow builds and pushes a odh-dashboard image for the specified platforms.
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    pre:
+      - ref: opendatahub-io-ci-s390x-build-create
+    test:
+      - ref: opendatahub-io-ci-s390x-build-dashboard
+    post:
+      - ref: opendatahub-io-ci-s390x-build-delete
+    env:
+      IC_CLI_VERSION: '2.23.0'
```

Comment:
Can you provide more information regarding this? I don't underestand why are we using that exact version

Reply:
There is no specific reason for specific version, I tested these scripts manually using the mentioned version, anyway this is going to be temporary approach until we have s390x cluster profile.